public/css/style.css

```diff
@@ -2,5 +2,5 @@
 	margin-left: 410px;
 }
 #sign-in{
```

Comment:
consider using a little white space after your selector and before the curly brace for readability